DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestExtensions/TestUserMapper.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using NSubstitute;
+using GitHubExtension.Security.StorageModels.Identity;
+using GitHubExtension.Security.WebApi.Library.Mappers;
+using GitHubExtension.Models.CommunicationModels;
+
+
+namespace GitHubExtension.Security.Tests.TestExtensions
+{
+    public class TestUserMapper
```

Comment:
Please revert class name, method conventions to using word "Test" in postfix form. For example, UserManager**Tests**, or TestToUserReturnModel -> EntinyToUserReturnModelTest and etc...